src/NHibernate/Mapping/Table.cs

```diff
@@ -27,7 +27,7 @@ public enum SchemaAction
 	public class Table : IRelationalModel
 	{
 		private readonly List<string> checkConstraints = new List<string>();
-		private readonly LinkedHashMap<string, Column> columns = new LinkedHashMap<string, Column>();
+		private readonly Dictionary<string, Column> columns = new Dictionary<string, Column>();
```

Comment:
I think it does make sense to have a linked hash map here: it ensures the order of elements. And the application does depend on the order of columns.